### PR TITLE
s/const/var so it works in old safari and ie11

### PR DESCRIPTION
**Pull request by ryankshaw**
in order to use this in canvas we have to make sure all the code is es5, which means we need to use 'var' instead of 'const'

### CODE REVIEW

**Review by a-b-r-o-w-n — DISMISSED** [review]

**Comment by ryankshaw** [comment]
can you merge this? I don't have write access

**Comment by ryankshaw** [comment]
bump, @astephenb or any other maintainer can you merge this? I don't have write access to this repo so I can't myself

**Comment by a-b-r-o-w-n** [comment]
@ryankshaw would you mind running a build with this change? I'll merge after. Thanks.

**Review by a-b-r-o-w-n — APPROVED** [review]

**Pull request merged**
